data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -401,7 +433,7 @@ def _workforce_factor(self) -> bool:
                     field_names.UNEMPLOYMENT_FIELD
                     + field_names.PERCENTILE_FIELD_SUFFIX
                 ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
```

Comment:
Nit: merge conflicts. I think you want this to go back to `>=`, here and everywhere.